test/testutil/driver.c

```diff
@@ -121,11 +121,12 @@ static void finalize(int success)
         ERR_print_errors_cb(openssl_error_cb, NULL);
 }
 
-static const char *test_title = NULL;
+static char *test_title = NULL;
 
 void set_test_title(const char *title)
 {
-    test_title = title;
+    free(test_title);
+    test_title = title == NULL ? NULL : strdup(title);
 }
```

Comment:
Just the framework or tests too? There are quite a few cases where we use OPENSSL_malloc in individual tests.